apps/site/assets/js/support-form.js

```diff
@@ -407,3 +408,17 @@ function showExpandedForm($) {
 function hideExpandedForm($) {
   $(".support-form-expanded").hide();
 }
+
+// Click on the blocks that have the 'expand' class (that serves as a flag) to expand them
+export function setupInitialStateOfExpandableBlocks($) {
```

Comment:
Are you sure this function is needed? The Bootstrap-included Javascript should essentially do this on page load, as long as the trigger link has `[data-toggle="collapse"]` (which it does!)